MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,447 @@
+        this.Content = dataForm;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+    
+    using Syncfusion.Maui.Core.Hosting;
+
+    public static class MauiProgram
+    {
+	    public static MauiApp CreateMauiApp()
+	    {
+	        var builder = MauiApp.CreateBuilder();
+		    builder
+			    .ConfigureSyncfusionCore()
+			    .UseMauiApp<App>()
+			    .ConfigureFonts(fonts =>
+			    {
+				    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+				    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+			    });
+
+		    return builder.Build();
+	    }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Creating data object
+
+The `SfDataForm` is a data edit control so, create a data object with details to create a data form based on your business requirement.
+
+Here, the data object named **DataModel** created with some properties.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+
+    public class ContactsInfo
+    {
+        public string FirstName { get; set; }
+    
+        public string MiddleName { get; set; }
+    
+        public string LastName { get; set; }
+    
+        public string ContactNumber { get; set; }
+    
+        public string Email { get; set; }
+   
+        public string Address { get; set; }
+
+        public DateTime? BirthDate { get; set; }
+    
+        public string GroupName { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+Initialize the data object in view model class in order to bind in 'DataObject' property of 'SfDataForm'.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public class DataFormViewModel
+    {
+        public DataModel DataModel {get; set;}
+        
+        public DataFormViewModel()
+        {
+            this.DataModel = new DataModel();
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set data object to data form
+
+By default, data form auto generate the editors based on the primitive data type in the 'DataObject' property. Please refer the following code to set the 'DataObject' property.
+
+{% tabs %}
+{% highlight MainPage.xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+
+                    <ContentPage.BindingContext>
+                        <local:DataFormViewModel/>
+                    </ContentPage.BindingContext>    
+	                        <dataForm:SfDataForm x:Name="dataForm" 
+						    DataObject="{Binding DataModel}"/>    
```

Comment:
DataObject="{Binding DataModel}" => DataObject="{Binding ContactsInfo}"

Reply:
DataModel to ContactsInfo changed